### PR TITLE
remove indicators when data report removed

### PR DESCRIPTION
**Pull request by gr2m**
closes https://github.com/eHealthAfrica/kazana/issues/79
requires https://github.com/eHealthAfrica/kazana/pull/116

### CODE REVIEW

**Comment by patriciagarcia** [comment]
LGTM. @obdulia-losantos, could you  please have a look again to see if it solves the issue?

**Comment by obdulia-losantos** [comment]
It solves the issue but know we have other... all the "transformers" execute the transform process

```
Info log 2015-11-05T10:31:06.020Z [ 'plugin-transformer-0', 'transformer-3', 'id-25' ] removed 796 integrated data docs ...
Info log 2015-11-05T10:31:06.149Z [ 'plugin-transformer-1', 'transformer-3', 'id-25' ] removed 796 integrated data docs ...
Info log 2015-11-05T10:31:06.216Z [ 'plugin-transformer-2', 'transformer-3', 'id-25' ] removed 796 integrated data docs ...
Info log 2015-11-05T10:31:06.271Z [ 'plugin-transformer-3', 'transformer-3', 'id-25' ] removed 796 integrated data docs ...
```

**Comment by gr2m** [comment]
@obdulia-losantos I've updated the PR, it now requires eHealthAfrica/kazana#116, too

@jo does that work for you?

File: lib/compose-plugin.js [hunk 1]
```diff
@@ -236,6 +232,8 @@ function handleDataReport (server, plugin, indicatorName, transform, report) {
       server.log([plugin.name, report.sourceId, report.scope.id, 'info'], 'removed ' + dataReports.length + ' integrated data docs ...')
 
       if (report._deleted) {
+        // TOOD: we have to remove the .transformedAt property from the submission doc,
+        //       otherwise it will keep picking it up on startup.
```
Comment (gr2m) [inline lib/compose-plugin.js]: if we are good with this PR, let's create an issue for this one before we merge

**Comment by obdulia-losantos** [comment]
:+1:

Now it works perfect :dart:

File: lib/compose-plugin.js (hunk 1 above)
Comment (patriciagarcia) [inline lib/compose-plugin.js]: @gr2m, not sure I understand where to remove `transformedAt`. Can you create the issue please?

Comment (gr2m) [inline lib/compose-plugin.js]: issue here, in terms what we want to achieve, not how: https://github.com/eHealthAfrica/kazana-server/issues/76

**Pull request merged**